TombEngine/Objects/TR3/Entity/tr3_civvy.cpp

```diff
@@ -2,418 +2,594 @@
 #include "Objects/TR3/Entity/tr3_civvy.h"
 
 #include "Game/control/box.h"
+#include "Game/control/lot.h"
 #include "Game/effects/effects.h"
-#include "Game/Lara/lara.h"
-#include "Game/itemdata/creature_info.h"
 #include "Game/items.h"
+#include "Game/itemdata/creature_info.h"
+#include "Game/Lara/lara.h"
 #include "Game/misc.h"
 #include "Game/people.h"
+#include "Math/Random.h"
```

Comment:
Since `Math.h` is already included, `Random.h` is not necessary. Math components were built kind of like a library, so all you need is the one header file. :)

Reply:
Done